src/content/reducers/profile-view.js

```diff
@@ -53,8 +59,8 @@ function profile(state: Profile | null = null, action: Action) {
       return Object.assign({}, state, { threads });
     }
     case 'ASSIGN_TASK_TRACER_NAMES': {
-      if (state === null) {
-        return null;
+      if (!state.tasktracer.taskTable.length) {
```

Comment:
once we'll have a structured type for the tasktracer structure, this will be safe. Currently I don't know...